tao_compiler/mlir/disc/tools/disc-replay/tar_helper.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DISC_REPLAY_TAR_HELPER_H_
+#define DISC_REPLAY_TAR_HELPER_H_
+
+#include <sstream>
+
+#include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/platform/subprocess.h"
+
+namespace replay {
+
+tensorflow::Status DeCompressTar(const std::string& tar_fname,
+                                 const std::string& out_dir) {
+  // tar xf <tar_fname> -C <out_dir>
+  tensorflow::SubProcess proc;
+  std::string stdout;
+  std::string stderr;
+  std::vector<std::string> args = {"tar", "xf", tar_fname,
+                                   "-C " + out_dir + "/"};
+  proc.SetProgram("/bin/tar", {"tar", "xf", tar_fname, "-C", out_dir});
```

Comment:
NIT. Not sure whether `tar` is installed in popular OS by default.

Reply:
`tar` is installed in BladeDISC runtime Docker, and I also add a `CheckTarCommand` function to check the `tar` command.